src/components/views/rooms/MemberInfo.js

```diff
@@ -48,6 +48,8 @@ import SettingsStore from "../../../settings/SettingsStore";
 import E2EIcon from "./E2EIcon";
 import AutoHideScrollbar from "../../structures/AutoHideScrollbar";
 import MatrixClientPeg from "../../../MatrixClientPeg";
+import Matrix from "matrix-js-sdk";
```

Comment:
Could we use the following instead?

```suggestion
import { EventTimeline } from "matrix-js-sdk";
```